### ISSUE TITLE
Profile several event types

### ISSUE THREAD

**Issue by marcelja** [jvm-profiling-tools/async-profiler#357]
Is it possible to profile two or more event types at the same time? Let's say I want to record `cpu` and `alloc` of one PID. I tried running the two commands in parallel:
```
./profiler.sh -d 10 -e cpu -f cpu.svg 12345
./profiler.sh -d 10 -e alloc -f alloc.svg 12345
```
But this would result in the same .svg file. Either `cpu` or `alloc` profile depending on what command was started first.

**Comment by krzysztofslusarski** [comment]
I believe @apangin has it on his roadmap:
https://github.com/jvm-profiling-tools/async-profiler/issues/150

**Comment by marcelja** [comment]
Ah, great, thanks!

**Comment by apangin** [comment]
Right. This is already implemented in [v2.0](https://github.com/jvm-profiling-tools/async-profiler/tree/v2.0) branch - it's possible to specify `-e cpu,alloc` together. But only `jfr` output is supported so far. Other formats will come soon.
